src/intl/messages.js

```diff
@@ -394,8 +394,8 @@ export const messages: { [messageId: string]: MessageType } = {
     message: 'Refresh',
     description: 'Reload data from server',
   },
-  remoteDesktopConnectionSettings: 'Remote Desktop Connection Settings',
-  remoteDesktopViewerConnection: 'Remote Desctop Viewer Connection',
+  remoteDesktopConnectionInstructions: 'Console Instructions',
+  remoteDesktopViewerConnection: 'Remote Desktop Viewer Connection',
   remoteDesktopViewerAvailable: 'Remote Desktop Viewer is available for most operating systems. To install it, search for it in GNOME Software or run the following:',
```

Comment:
rename `remoteViewerAvailable`

The thick client's name is `Remote Viewer` (no `Desktop`) so remove `Desktop`
(The term `Remote Desktop` is easy to confuse with MS RDP, so I want to be really careful)